wagtail/admin/templates/wagtailadmin/panels/inline_panel.html

```diff
@@ -15,40 +15,22 @@
 {% endif %}
 
 <div id="id_{{ self.formset.prefix }}-FORMS">
```

Comment:
Why not put the `data-w-formset-target="forms"` & `data-w-orderable-target="container"` here?

```suggestion
<div id="id_{{ self.formset.prefix }}-FORMS" data-w-formset-target="forms" data-w-orderable-target="container">
```

And keep this as the parent of the inline panel children as per the current code?

Remembering that the `inline_panel.html` will have the `data-controller=...` on it, so we can still have the mostly existing DOM structure in this template.

Maybe review these changes and see if there's a cleaner way without such a large diff to this file. I am pretty sure it was solved in our call by setting the `data-controller` on the outer `InlinePanel` class in Python.